server/node_test.go

```diff
@@ -121,6 +121,7 @@ func TestBootstrapCluster(t *testing.T) {
 	for _, kv := range rows {
 		keys = append(keys, kv.Key)
 	}
+	// TODO(marc): this depends on the sql system objects.
```

Comment:
Yeah, this is annoying. Maybe we should strip keys that start with `TableDataPrefix`.